sel4test-sim/builds.yml

```diff
@@ -28,13 +29,20 @@ variants:
 #     smp: ['', SMP]
 #     hyp: ['', hyp]
 #     mcs: ['', MCS]
+#     domains: ['', DOM]
 #     compiler: [gcc, clang]
 #     mode: [32, 64]
 
 
 # only generate builds for platofrms that have `simulation_binary` set
+# only generate builds with DOM set if debug is also set
```

Comment:
In another commit you have:
```
# run NUM_DOMAINS > 1 tests only on release builds
```
That would mean NUM_DOMAINS > 1 implies release which is not debug? Would that set DOM? Bit unclear on that.

Reply:
in sel4test-sim we pick debug builds, in sel4test-hw we pick release builds. More overall coverage without increasing number of tests too much.